unittests/util/MockCache.h

```diff
@@ -25,7 +25,14 @@
 
 struct MockCache
 {
-    MOCK_METHOD(void, update, (std::vector<data::LedgerObject> const& a, uint32_t b, bool c), ());
+    MOCK_METHOD(void, updateImp, (std::vector<data::LedgerObject> const& a, uint32_t b, bool c), ());
+
+    // mock update has a parameter with default value
```

Comment:
This comment does not seem very useful as we can clearly see it from the code. Consider removing 👍